src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -249,23 +263,27 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
                 fullWidth
                 labelId="visiblityConditionLabel21"
                 label={ strings.draftEditorScreen.editor.visibility.conditionalFieldValue }
-                value={ requiredFieldBoolean }
+                value={ requiredConditionFieldBoolean }
                 onChange={ event => updateFormFieldVisiblityValues({
                   ...field,
                   visibleIf: {
-                    field: requiredField,
+                    field: requiredConditionField,
                     equals: event.target.value,
                     notEquals: "",
                     and: [],
                     or: []
                   }
-                }, event.target.value) }
+                }, event.target.value)
+                }
               >
+                <MenuItem sx={{ color: "gray" }}>Valitse kenttä</MenuItem>
                 <MenuItem value="kylla">{ strings.generic.yes }</MenuItem>
                 <MenuItem value="ei">{ strings.generic.no }</MenuItem>
               </Select>
             </FormControl>
-            <Divider/>
+            <Typography sx={{ color: "gray", visibility: requiredConditionInfoField }}>
```

Comment:
Instead of toggling css visiblity property, could this be rendered conditionally?
Something like:
``` suggestion
          { requiredConditionInfoField
            ? <Typography sx={{ color: "gray" }}>{ strings.draftEditorScreen.editor.visibility.conditionalFieldValueInfo }</Typography>
            : null
          }
```